src/main/scala/io/github/spark_redshift_community/spark/redshift/DefaultSource.scala

```diff
@@ -97,7 +97,7 @@ class DefaultSource(
     }
 
     def tableExists: Boolean = {
-      val conn = jdbcWrapper.getConnector(params.jdbcDriver, params.jdbcUrl, params.credentials)
```

Comment:
I'm curious, did you look into the history of why they ignore all the other params?

Even on latest master, only use these 3 params and ignore the others https://github.com/spark-redshift-community/spark-redshift/blob/master/src/main/scala/io/github/spark_redshift_community/spark/redshift/RedshiftRelation.scala#L111

Reply:
Looks like they are fixing the same `invalid character` 8 years ago like we were last week 😅 , the way they fix is passing user/password as part of the options in this [pr](https://github.com/databricks/spark-redshift/issues/132#issuecomment-174050108)